tasks/system_probe.py

```diff
@@ -26,7 +26,7 @@
 DNF_TAG = "dnf"
 
 CLANG_CMD = "clang {flags} -c '{c_file}' -o '{bc_file}'"
-LLC_CMD = "llc -march=bpf -filetype=obj -o '{obj_file}' '{bc_file}'"
+LLC_CMD = "llc -march=bpf -mcpu=v2 -filetype=obj -o '{obj_file}' '{bc_file}'"
```

Comment:
I know you are just testing things at this point, but NPM cannot use this for the prebuilt objects since it runs on kernels < 4.14.

Reply:
oh that's a good point !